### ISSUE TITLE
Include BoomPow GUI

### ISSUE THREAD

**Issue by mrexodia** [BananoCoin/boompow-legacy#17]
Yesterday I created [BoomPowGui](https://github.com/mrexodia/BoomPowGui), which includes a pre-installed embedded python, minimizes to the system tray and has an option to start on user login:

![BoomPowGui](https://github.com/mrexodia/BoomPowGui/raw/master/BoomPowGui/screenshot.png)

**Comment by MattFaz** [comment]
Looks awesome, running well on my Win10 machine. Thanks for creating!

**Comment by mrexodia** [comment]
![give_ban](https://user-images.githubusercontent.com/2458265/118731822-804b5780-b839-11eb-989c-efefd99dcc47.png)

**Comment by patrickmostert** [comment]
Hey @mrexodia ,
I see this in the GUI. is that good?
see screenshot

![download (1)](https://user-images.githubusercontent.com/84547484/119060035-b7547100-b9d1-11eb-93e2-36b500cc1353.png)

**Comment by mrexodia** [comment]
Yep, that’s just the work service doing work. If you see any errors in the
bpow-client you can just stop and start again (sometimes it naturally
happens if the server is down or something)

**Comment by patrickmostert** [comment]
@mrexodia Okay, thank you!

Approximately how long does it take to receive the first transaction?

**Comment by mrexodia** [comment]
I get paid usually around 10AM every day, but the last few days I didn’t
🤷🏻‍♂️
